### PR TITLE
add demo page in Demo package

### PR DESCRIPTION
**Pull request by Vivek1012**

### CODE REVIEW

**Comment by jkinable** [comment]
As per my request, Vivek Talreja took some of the demo classes from our wiki and added them to our demo package. For some reason, we only had these classes in our wiki, but not in the demo package. The next step would be to figure out whether we can inline classes/snippets from our demo package on our wiki. That way, we don't need to update both our demo package and wiki pages whenever we make changes to the library.

File: jgrapht-demo/src/main/java/org/jgrapht/demo/DirectedGraphDemo.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
+/* 
+ * Copyright (C) 2008-2018,  Minh Van Nguyen &lt; nguyenminh2@gmail.com&gt;
```
Comment (jkinable) [inline jgrapht-demo/src/main/java/org/jgrapht/demo/DirectedGraphDemo.java]: Can you replace this old class header by the new one? Just copy-paste the class header from another class, and change the year to 2008-2018 and give the copyright to Minh Van Nguyen

File: jgrapht-demo/src/main/java/org/jgrapht/demo/DirectedGraphDemo.java [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    /**
+     * The starting point for the demo.
+     *
+     * @param args ignored.
+     */
+    
+    public static void main(String args[])
+    {
+        // constructs a directed graph with the specified vertices and edges
+        DefaultDirectedGraph<String, DefaultEdge> directedGraph =
```
Comment (jkinable) [inline jgrapht-demo/src/main/java/org/jgrapht/demo/DirectedGraphDemo.java]: Make this Graph<String, DefaultEdge> directedGraph =

File: jgrapht-demo/src/main/java/org/jgrapht/demo/LabeledEdges.java [hunk 3]
```diff
@@ -0,0 +1,134 @@
+/*
+ * (C) Copyright 2012-2018, by Oliver Schrenk and Contributors.
```
Comment (jkinable) [inline jgrapht-demo/src/main/java/org/jgrapht/demo/LabeledEdges.java]: It seems I was mistaken. Can you change the copyright of this class to Barak Naveh

**Comment by Vivek1012** [comment]
Done with the minor changes.

**Comment by jkinable** [comment]
Looks good, thanks!
At a later point in time, we would need to revise the Demo classes. Currently the wiki just links to 5 random, rather unrelated demo classes. That is not a nice way to learn how to use JGraphT. Having a proper tutorial, with different, related demo's would be desirable. Also we need to make sure that the Demo classes cover various aspects of JGraphT.

**Pull request merged**